ablySpec/RestClient.stats.swift

```diff
@@ -15,16 +15,14 @@ import Foundation
 private func postTestStats(stats: JSON) -> ARTClientOptions {
     let options = AblyTests.setupOptions(AblyTests.jsonRestOptions);
     
-    let key = ("\(options.key)" as NSString)
-        .dataUsingEncoding(NSUTF8StringEncoding)!
-        .base64EncodedStringWithOptions(NSDataBase64EncodingOptions(rawValue: 0))
+    let keyBase64 = encodeBase64(options.key ?? "")
     
-    let request = NSMutableURLRequest(URL: NSURL(string: "https://rest.ably.io/stats")!)
+    let request = NSMutableURLRequest(URL: NSURL(string: "https://sandbox-rest.ably.io/stats")!)
```

Comment:
Yes, but surely the environment shouldn't be hard coded in and instead we should get the domain based on the environment you are testing against?